src/components/Common/VideoCard/VideoOptions.tsx

```diff
@@ -54,7 +54,11 @@ const VideoOptions = ({
       hideVideo({ variables: { request: { publicationId: video?.id } } })
     }
   }
-
+  function onClickWatchLater() {
```

Comment:
can be a arrow function to follow uniformity and line spaces above and below function is appreciated 🙇🏻

Reply:
Sorry I'll change it right away